pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -96,8 +96,13 @@ const (
 
 var isValidExpression = regexp.MustCompile(`^[A-Za-z0-9_.+-]+$`).MatchString
 
+type NetworkValidator interface {
```

Comment:
Because this interface is used to describe the required behavior expected by `VMICreateAdmitter`, there is no need to export it:
```suggestion
type networkValidator interface {
```

Reply:
Done